packages/adminpanel/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "alvtime-admin-react-pwa",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://www.alvtime.no/adminpanel",
+  "homepage": ".",
```

Comment:
Is there anyway we could get this change into master without breaking the current setup?